libc/utils/gpu/loader/amdgpu/Loader.cpp

```diff
@@ -326,6 +326,11 @@ static hsa_status_t hsa_memcpy(void *dst, hsa_agent_t dst_agent,
   return HSA_STATUS_SUCCESS;
 }
 
+void print_resources(void *image) {
+  fprintf(stderr, "Printing resource usage on AMDGPU is not supported yet.\n");
+  exit(EXIT_FAILURE);
```

Comment:
Don't make it fail, it'll just make it more annoying when we try to run benchmarks on both.